tests/test_installed_distributions.py

```diff
@@ -65,3 +66,28 @@ def mock_call(*args, cwd=None):
     monkeypatch.setattr(pip_api._installed_distributions, "call", mock_call)
 
     _ = pip_api.installed_distributions(local=True)
+
+
+@pytest.mark.skipif(
+    pip_api.PIP_VERSION < parse("19.2"),
+    reason="Pip version too old to support --path parameter",
+)
```

Comment:
Would prefer to test that this fails correctly instead of skipping. I do something similar at https://github.com/di/pip-api/blob/master/pip_api/_hash.py, although that won't work directly here.